calico/felix/felix.py

```diff
@@ -411,9 +393,7 @@ def handle_endpointdestroyed(self, message, sock):
             return
 
         # Unsubscribe endpoint.
-        sock = self.sockets[Socket.TYPE_ACL_SUB]
-        sock._zmq.setsockopt(zmq.UNSUBSCRIBE, delete_id.encode('utf-8'))
-
+        self.sockets[Socket.TYPE_ACL_SUB].unsubscribe(delete_id.encode('utf-8'))
```

Comment:
We should check the other handle_* functions to see if they make the same mistake.